gradle/libs.versions.toml

```diff
@@ -28,9 +28,9 @@ adapters = "1.10-SNAPSHOT"
 commodore = "2.2"
 bungeecord = "a7c6ede"
 velocity = "3.1.1"
-fabric-minecraft = "1.20"
+fabric-minecraft = "1.20.2"
```

Comment:
Is this necessary? And if we bump the dependencies, shouldn't we bump the required versions in the mod json file?

Reply:
Considering 1.20/1.20.1 Via* don't exist & Geyser will error when trying to run natively, I think it's better to bump it so the mod doesn't start at all. I'll adjust the mod file too